### PR TITLE
Update Repository Readme

### LINKED ISSUE #3177: Make translation instructions easier to find
Currently, the only locations where a link to instructions about how to help translate are the "Help Translate" button in the Scratch Addons settings and within the docs themselves. This makes it relatively hard to find and even harder to stumble across.

A couple of potential places to put it:
- Under the language dropdown on scratchaddons.com
- In the description for "International" (under the "Features" header) on scratchaddons.com
- In the footer on scratchaddons.com
- In the "Contribute" section of the README in the ScratchAddons repo
- In the "Contributing guidelines" (`CONTRIBUTING.md`) in the ScratchAddons repo
- Some/all of the above

### LINKED ISSUE #5411: Remove mention of persistent scripts from README
I know that Manifest V3 is going to make persistent scripts harder to manage, and that you removed the docs page on creating them earlier this year, but they're still mentioned in this repository's `README.md`. Should we remove the mention of them there too?

### PR DESCRIPTION
**Pull request by Samq64**
Resolves #3177
Resolves #5411
Closes #5603

Previous PR: #6208

### Changes

Makes some significant changes to the repo README.

### Reason for changes

The README needs an update.

### Tests

https://github.com/Samq64/ScratchAddons/tree/update-readme2#readme

### CODE REVIEW

**Comment by apple502j** [comment]
Images should be placed inside `.github`.

**Comment by Samq64** [comment]
I think the content looks really good now. I just need to find a way to make the top look a little better. The problem with moving the installation to the top is I want to keep the entire about section together and I think it's more important than the installation instructions, but I also want to keep the badges on top for quick access for users who just want to install the store versions. Maybe the second row of badges could be removed though.

**Comment by Samq64** [comment]
I added the pending label because the installation instructions mention building the settings page now.

File: README.md
Comment (CST1229) [inline README.md:18]:
```suggestion
Scratch Addons combines new and existing features and themes for the [Scratch](https://scratch.mit.edu) website and project editor into one easy-to-access and configurable browser extension. The mission is to provide a centralized, up-to-date platform for community development of new features and themes for Scratch.
```

Comment (WorldLanguages) [reply README.md:18]: Maybe https://scratch.mit.edu/about if you prefer.

File: README.md
Comment (WorldLanguages) [inline README.md:30]: I must admit the `addon.* APIs` description on the docs is quite old. Only one addon sends notifications and not through an addon.* API.

```suggestion
Userscripts can also use the `addon.*` JavaScript APIs provided by Scratch Addons, which make it easy to obtain Scratch-related information like the currently logged in user, wait until a certain element exists on the page, or get a reference to the Scratch VM object.
```

File: README.md
Comment (WorldLanguages) [inline README.md:36]: I would remove this whole section until we have better docs on non-addon code, in that case we'll just link to them.

File: README.md
Comment (WorldLanguages) [inline README.md:26]:
```suggestion
### About addons
```

File: README.md
Comment (WorldLanguages) [reply README.md:36]: Maybe this section could be renamed to `About the browser extension` and it can be made shorter.

File: README.md
Comment (WorldLanguages) [inline README.md:46]: Maybe this could be rephrased? Something like:

```suggestion
### Addons beyond the Scratch Addons browser extension
```

File: README.md
Comment (WorldLanguages) [reply README.md:30]: Even better if you modify my suggestion to imply the addon.* API can do many more things than the three ones I mentioned.

File: README.md
Comment (WorldLanguages) [reply README.md:36]: Maybe we could go straight to the point: the browser extension only runs addons when necessary (if the URL matches), it provides addon.* APIs and provides a settings page to the user.

**Comment by WorldLanguages** [comment]
In the `What is an addon?` section (or `About addons`), we could mention some basic values that we take for granted: addons should be compatible with each other, be optimized to avoid hurting performance (specially in the editor), be localizable, accessible (which includes color contrast and RTL support), have good browser support, preserve user privacy, etc.
We should also mention addon manifests, not just the `addon.* APIs`.

File: README.md
Comment (WorldLanguages) [inline README.md:28]:
```suggestion
An addon is one or more [userscripts](https://scratchaddons.com/docs/develop/userscripts/) (written in JavaScript) or [userstyles](https://scratchaddons.com/docs/develop/userstyles/) (written in CSS) that run on the Scratch website or project editor.
```

File: README.md
Comment (WorldLanguages) [inline README.md:48]: We could mention adacraft (TW fork)

Comment (Samq64) [reply README.md:48]: We link to the source for TurboWarp. Should we just to the site or editor for both?

File: README.md
Comment (DNin01) [inline README.md:5]: Looks like both of these links go to the Chrome Web Store.
```suggestion
[![Firefox Add-ons](.github/readme-images/ff-addon-badge.png)](https://addons.mozilla.org/firefox/addon/scratch-messaging-extension/)
```

**Review by DNin01 — DISMISSED** [review]

**Review by DNin01 — COMMENTED** [review]
Seems okay to me. I'll review more thoroughly before I approve.

File: README.md
Comment (Hans5958) [inline README.md:2]: For the not-sure-how-many times here, I would still suggest this.

```suggestion
# Scratch Addons
```

**Review by Hans5958 — APPROVED** [review]

File: README.md
Comment (Hans5958) [reply README.md]: Just want to tell that I don't intend malice or being forceful here. If you think that's the case, I don't mean to. Sorry.

Comment (Samq64) [reply README.md]: I didn't mean to sound rude.

Comment (WorldLanguages) [reply README.md]: It's okay, if someone cares enough about changing the title they can make a dedicated PR for that after we merge this one

File: README.md
Comment (DNin01) [inline README.md:40]: Might I suggest rewording the part where a new version is released at least once a month? I would also use the word "release" and mention how each major update releases with new features.

Comment (WorldLanguages) [reply README.md:40]: Go ahead

**Review by DNin01 — APPROVED** [review]
All well written. Good job!

**Comment by WorldLanguages** [comment]
This should be pretty good to go if we remove the parts that mention building the `webpages` folder, as most likely we'll merge this first.

**Review by WorldLanguages — APPROVED** [review]
🎉

File: README.md
Comment (apple502j) [inline README.md:59]: Here as well

File: README.md
Comment (apple502j) [inline README.md:65]: We do not build webpages for now.

File: README.md
Comment (apple502j) [inline README.md:79]: oxford comma

**Review by apple502j — CHANGES_REQUESTED** [review]

File: README.md
Comment (WorldLanguages) [reply README.md]: I may try to rewrite this but keeping the content scripts parallelism.

**Review by apple502j — APPROVED** [review]

**Pull request merged**
